ReactNativeClient/lib/services/ExternalEditWatcher.js

```diff
@@ -237,12 +221,61 @@ class ExternalEditWatcher {
 					clearInterval(iid);
 					reject(wrapError(error));
 				});
+				if (options.onClose) {
+					subProcess.on('close', options.onClose);
+				}
 			} catch (error) {
 				throw wrapError(error);
 			}
 		});
 	}
 
+	async spawnDefault(file, onClose) {
+		// windows
+		let path = '';
+
+		if (shim.isWindows()) {
+			path = 'start';
+		} else if (shim.isMac()) {
+			path = 'open';
+		} else if (shim.isLinux() || shim.isFreeBSD()) {
+			path = 'xdg-open';
+		} else {
+			// Fallback on the electron open method
+			return new Promise((resolve, reject) => {
+
+				if (bridge().openExternal(`file://${file}`)) {
+					resolve();
+				} else {
+					reject(new Error('Unknown failure'));
```

Comment:
Please could you change the error to `'Could not open file: ' + file`. Sometimes these messages end up in logs so it's good to have a bit more context.

Reply:
Done!